### PR TITLE
Update to allow testing against Sauce Labs and Browserstack

### PR DESCRIPTION
**Pull request by michael-vansickle**
Updated Gruntfile and intern configurations to allow tests to run against saucelabs and browserstack via grunt command line settings

### CODE REVIEW

**Comment by kitsonk** [comment]
Can you switch over to SauceLabs before I merge this one?

File: typings/browser.d.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+/// <reference path="browser/ambient/jsdom/index.d.ts" />
```
Comment (kitsonk) [inline typings/browser.d.ts]: Why is this here, it seems tangental to the issue being addressed.

**Comment by michael-vansickle** [comment]
I've updated the .travis.yml file to use Saucelabs. I also cleaned out the old jsdom stuff that made its way in when I was experimenting with something.

**Comment by kitsonk** [comment]
Merged in ee8949753f562abab17fed29717609c16703a947
